addOns/pscanrulesAlpha/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Added
+
+ - Add rules for Cross-Origin-Resource-Policy
```

Comment:
This should be singular, only one rule is being added.

Reply:
Should also have a period, and don't need a leading blank line (it can come right after the Added header).